skfuzzy/fuzzymath/tests/test_fuzzy_ops.py

```diff
@@ -312,6 +312,19 @@ def test_interp_membership():
     assert_allclose(np.r_[yy], np.r_[0.641])
 
 
+def test_interp_value():
+    x = np.r_[0:4.1:0.1]
+    mfx = trapmf(x, [0, 1, 2, 4])
+
+    xx = interp_value(x, mfx, 0.5)
+    assert_allclose(xx, [0.5,3])
+
+    xx = interp_value(x, mfx, 0.0)
+    assert_allclose(xx, [0,4])
+
+    xx = interp_value(x, mfx, 1.5)
+    assert len(xx) == 0
+
```

Comment:
A neat test to add is that all result(s) from `result = interp_value(x, mfx, val)` should equal within tolerance `val` when `interp_membership(x, mfx, result)` is called. Essentially a roundtrip through these functions should (and does) succeed.